roles/deploy_collection/tasks/configure-ocp.yml

```diff
@@ -28,7 +26,7 @@
         - {{ deploy_collection_vm_ssh_public_key }}
 
 - name: Create cloud-init Secret
-  kubernetes.core.k8s:
+  kubevirt.core.kubevirt_vm:
```

Comment:
`kubevirt.core.kubevirt_vm` is basically a replacement for calls to `kubernetes.core.k8s` with `kind` set to `VirtualMachine`. Also, in case you are using the certified collection, the module has the following FQCN: `redhat.openshift_virtualization.kubevirt_vm`.